specs/sql/README.md

```diff
@@ -896,29 +917,117 @@ right-hand dataset of $N_r$ rows of $M_r$ columns, then the cartesian
 product is a dataset of $N_l \times N_r$ rows, each
```

Comment:
I realized this is not being rendered correctly:

<img width="1017" alt="Screen Shot 2023-01-05 at 10 44 26" src="https://user-images.githubusercontent.com/1233473/210794415-ee06c1e1-aad5-4c13-b3c5-a38b5d6ca2ca.png">

Reply:
On GitHub?

Reply:
Yeah. See: https://github.com/tablelandnetwork/docs/tree/main/specs/sql#join-clause

Reply:
Ok, no idea why that was the case. But after a bunch of attempts to fix, I have it working. See https://github.com/tablelandnetwork/docs/blob/a6a636bee10d8d3242026ed77f13dd0b412f640b/specs/sql/README.md#details-9. I ended up just re-writing that "by hand" again, and it works. So there must have been some sort of special utf-8 char in there or something messing things up?